site/kube-addon.md

```diff
@@ -65,6 +65,17 @@ Kubernetes see the
 and the [NetworkPolicy API object
 definition](http://kubernetes.io/docs/api-reference/extensions/v1beta1/definitions/#_v1beta1_networkpolicy).
 
+### Dropped connections
+
+Details on connections dropped by the Network Policy Controller can be found in the `weave-npc` container's logs:
```

Comment:
I wonder if this should start by explaining why you might need to look there, e.g. "If you suspect that legitimate traffic is being blocked by the Weave Network Policy Controller ..."